Utils/Dataflow/pyDKB/common/json_utils.py

```diff
@@ -30,10 +72,16 @@ def valueByKey(json_data, key):
 def nestedKeys(key):
     """ Transform STRING with nested keys into LIST.
 
-    Parameters:
+    **Parameters:**
         STRING key -- dot-separated list of nested keys.
-                      If a key contains dot itself, the key must be put between
-                      quotation marks.
+                      If a key contains dot itself, the key must be put
```

Comment:
Please do not forget that originally it was written with no concern to Sphinx, reStructuredText and building documentation from this comment. Thus newline in the comment does not mean newline or new paragraph in the built version — check how it looks now. Is it what you expected to get there?

Reply:
It means a newline in built version when [source] button is clicked.
![image](https://user-images.githubusercontent.com/11015074/45216317-d30e9c00-b2a8-11e8-8451-d59eb21ff8ad.png)
[docs] version is unaffected.

Reply:
Sorry, didn\`t mention that I was talking about PDF version.
HTML looks fine -- couldn\`t check it from tablet ;)